submissions/xT0PERx/Friends app/js/index.js

```diff
@@ -0,0 +1,137 @@
+const allSavedUsers = {
+  current: [],
+  changed: [],
+};
+const countUsers = 72;
+const usersBox = document.getElementById("usersList");
+const allFilters = document.getElementById("filters_menu");
+const searchInput = document.querySelector(".searcher");
+
+function getUsers() {
+  const url = `https://randomuser.me/api/1.3/?results=${countUsers}`;
+
+  return fetch(url)
+    .then((res) => {
+      if (res.ok) {
+        return res.json();
+      }
+      throw new Error(res.statusText);
+    })
+    .then((data) => data.results)
+    .catch((err) => {
+      console.error(err.message);
+    });
+}
+
+function initApp() {
+  getUsers().then((users) => {
+    saveUsers(users);
+    createUsersCards(allSavedUsers.changed);
+  });
+}
+
+initApp();
+
+function saveUsers(users) {
+  users.forEach(function (el) {
+    allSavedUsers.current.push(el);
+    allSavedUsers.changed.push(el);
+  });
+}
+
+function createUsersCards(users) {
+  const listsFragment = document.createDocumentFragment();
+
+  users.forEach(function (el) {
+    const card = document.createElement("li");
+    card.classList.add("userCard");
+    let template = `<span class="name">${el.name.first} ${el.name.last}</span>
+     <img src="${el.picture.large}">
+     <span class="age">Age: ${el.dob.age}</span>
+     <span class="location">Location: ${el.location.city}</span>`;
+    card.insertAdjacentHTML("beforeEnd", template);
+    listsFragment.append(card);
+  });
+
+  usersBox.appendChild(listsFragment);
+}
+
+allFilters.addEventListener("change", function ({ target }) {
+  let filteredUsers = null;
+
+  if (target.value === "reset") {
+    resetFilters();
+    return;
+  }
+
+  if (target.name === "search") {
+    return;
+  }
+
+  const sortedCases = {
+    sort_by_alphabet: (users) => nameSorters[target.value](users),
```

Comment:
I don't like `snake_cases`. In JS you should use `camelCase`. So you can rename `name` :)